src/test/java/org/codehaus/plexus/util/xml/pull/eduni_misc_Test_BjoernHoehrmannviaHST2013_09_18_Test.java

```diff
@@ -207,13 +206,15 @@ public void testhst_bh_006()
      * Version:
      *
      * @throws java.io.IOException if there is an I/O error
+     *
+     * NOTE: This test is SKIPPED as the MXParser object alone is unable to detect whether UTF-8 file
+     * has a BOM or not
      */
-    @Test
+    // @Test
```

Comment:
Since the PR deleted the encoding detection inside the MXParser, this test case (part or the standard xml 1.0 test suite) is not supported. That's why is skipped from execution.